octoprint_automation_scripts/__init__.py

```diff
@@ -332,12 +379,15 @@ def get_assets(self):
 
     def get_api_commands(self):
         return dict(
-            [(script_id, []) for script_id in self.scripts]
+            [(script_id, []) for script_id in self.scripts] +
+            [('cancel', [])]
         )
 
     def on_api_command(self, command, data):
         if command in self.scripts:
             self.start(command)
+        elif command == 'cancel':
+            self.relinquish_control(wait=False)
```

Comment:
Good point. Ill flip this and make a note in the documentation about `cancel` being reserved.